appservice/src/tree/FolderTreeItem.ts

```diff
@@ -10,23 +10,37 @@ import { ParsedSite } from '../SiteClient';
 import { ISiteFileMetadata, listFiles } from '../siteFiles';
 import { FileTreeItem } from './FileTreeItem';
 
+export interface FolderTreeItemOptions {
+    site: ParsedSite;
+    label: string;
+    path: string;
+    isReadOnly: boolean;
+    contextValuesToAdd?: string[];
+}
```

Comment:
Nice touch adding these options interfaces. Much safer than passing a bunch of parameters directly into the constructors 👍👍